File: front_yafiak_emergency_manager/src/js/class/Sensor.js
/**
 * @class Sensor.js
 * Describe what is a Sensor for App
 */

 class Sensor {

    /**
     * @param {string} id
     * @param {number} lx
     * @param {number} cy
     * @param {string} latitude
     * @param {string} longitude
     * @param {number} intensity 
     */

   constructor(id, lx, cy, latitude, longitude, intensity) {
      this.id = id;
      this.lx = lx;
      this.cy = cy;
      this.latitude = latitude;
      this.longitude = longitude;
      this.intensity = intensity;
   }

   getId() { return this.id; }
   getLx() { return this.lx}
   getCy() { return this.cy}
   getLatitude() { return this.latitude; }
   getLongitude() { return this.longitude; }
   getIntensity() { return this.intensity; }
 }

 export { Sensor };

File: YAFIAKSimulationApplication/src/main/java/com/yafiak/simulation/model/Sensor.java
package com.yafiak.simulation.model;

public class Sensor {

	private int x;
	private int y;
	private int intensity;
	private float latitude;
	private float longitude;
	
	public Sensor() {;}
	
	public Sensor(int x, int y, int intensity, float latitude, float longitude) {
		this.x = x;
		this.y = y;
		this.intensity = intensity;
		this.latitude = latitude;
		this.longitude = longitude;
	}

	public float getLatitude() {
		return latitude;
	}

	public void setLatitude(float latitude) {
		this.latitude = latitude;
	}

	public float getLongitude() {
		return longitude;
	}

	public void setLongitude(float longitude) {
		this.longitude = longitude;
	}

	public int getX() {
		return x;
	}

	public void setX(int x) {
		this.x = x;
	}

	public int getY() {
		return y;
	}

	public void setY(int y) {
		this.y = y;
	}

	public int getIntensity() {
		return intensity;
	}

	public void setIntensity(int intensity) {
		this.intensity = intensity;
	}
	
}
